bulk_indexer.go

```diff
@@ -396,7 +423,7 @@ func (b *BulkIndexer) Flush(ctx context.Context) (BulkIndexerResponseStat, error
 					if len(buf) == 0 {
 						n, err := gr.Read(buf[:cap(buf)])
 						if err != nil && err != io.EOF {
-							return resp, fmt.Errorf("failed to read from compressed buffer: %w", err)
+							return resp, fmt.Errorf("failed to read from uncompressed buffer: %w", err)
```

Comment:
I am a bit confused here, I think `compressed` is more correct? What was your reasoning to use `uncompressed`?

Reply:
Oops, I was mistaken here, it reads from the compressed buffer and we get uncompressed data. will revert the change.